dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/dispatcher/internal/channel/HttpDispatcherLink.java

```diff
@@ -462,10 +487,21 @@ private void wrapHandlerAndExecute(Runnable handler) {
                 taskWrapper.setClassifiedExecutor(classifyExecutor);
                 classifyExecutor.execute(taskWrapper);
             } else {
-                taskWrapper.run();
+                if (HttpDispatcher.getInterceptorValue() || handlerWC != null) {
```

Comment:
```suggestion
                if (handlerWC != null) {
```
if hasInterceptors returns true, but handlerWC equals null it won't work.